Resources/Prototypes/Catalog/Fills/Items/belt.yml

```diff
@@ -63,6 +63,20 @@
       - id: Ointment
         amount: 3
       - id: EmergencyMedipen #You never know what people are going to latejoin into
+      
+- type: entity
+  id: ClothingBeltParamedicFilled
+  parent: ClothingBeltMedical
+  suffix: Filled
+  components:
+  - type: StorageFill
+    contents:
+      - id: Brutepack
+        amount: 2
+      - id: Ointment
+        amount: 2
+      - id: EmergencyMedipen #You never know what people are going to latejoin into  
+        amount: 6
```

Comment:
This should be updated to include the bleed update item; blood pack, and maybe one stack of gauze?

Reply:
Absolutely, this PR is outdated.